src/qililab/qprogram/quantum_machines_compiler.py

```diff
@@ -322,14 +332,21 @@ def _handle_measure(
             if self._buses[element.bus].current_gain is not None
             else None
         )
+        rotation: float = (
```

Comment:
this is the only variable with a type should we remove it if it is causing a conflict?

Reply:
Yes you are right! I was testing some things regarding code quality and forgot to change it.